libdebug/cffi/ptrace_cffi_source.c

```diff
@@ -326,6 +679,7 @@ uint64_t ptrace_pokedata(int pid, uint64_t addr, uint64_t data)
     return ptrace(PTRACE_POKEDATA, pid, (void *)addr, data);
 }
 
+#ifdef ARCH_AMD64
 uint64_t ptrace_peekuser(int pid, uint64_t addr)
```

Comment:
only called by ptrace_interface.py. But, as already commented there, the caller is not useful anymore (no more hw bp management in python)

Reply:
same for poke_user